packages/module/patternfly-docs/content/examples/VirtualizedTable.md

```diff
@@ -8,11 +8,20 @@ propComponents: ['VirtualTableBody']
 
 Note: React Virtualized Extension lives in its own package at [`@patternfly/react-virtualized-extension`](https://www.npmjs.com/package/@patternfly/react-virtualized-extension)!
 
-
 import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
 import FilterIcon from '@patternfly/react-icons/dist/esm/icons/filter-icon';
 import { CellMeasurerCache, CellMeasurer} from 'react-virtualized';
 import { AutoSizer, VirtualTableBody, WindowScroller } from '@patternfly/react-virtualized-extension';
+import { Table as TableDeprecated, TableHeader as TableHeaderDeprecated } from '@patternfly/react-table/deprecated';
+import {
+  Dropdown,
```

Comment:
is it worth using the new dropdown here?

Reply:
I think that could be done as follow up? but since it's using the deprecated table anyway it didn't feel too bad.